source/jormungandr/tests/journey_common_tests.py

```diff
@@ -818,6 +902,8 @@ def test_journeys_without_show_codes(self):
             .format(from_sa="A", to_sa="D", datetime="20120614T080000")
 
         response = self.query_region(query, display=False)
+        check_best(response)
+        #self.is_valid_journey_response(response, query)# linestring with 1 value (0,0)
```

Comment:
I can't activate the check because the linestring contains only one (0,0) element. The check asserts that at least 2 elements is needed. I don't know if I should correct the check or correct the code. As that's only adding test, I let that like that for the future.